lib/webdriveragent.js

```diff
@@ -217,33 +217,21 @@ class WebDriverAgent {
     }
 
     let status = null;
-    let lastError = null;
     try {
       await waitForCondition(async () => {
         try {
           status = await getStatus();
           return true;
-        } catch (err) {
-          lastError = err;
-          return false;
-        }
+        } catch (ign) {}
+        return false;
       }, {
         waitMs: timeoutMs,
         intervalMs: 300,
       });
-    } catch (err) {
-      throw new Error(`WDA was not ready in ${timeoutMs} ms`);
-    }
-
-    if (status) {
-      return status;
+    } catch (ign) {
+      throw new Error(`WDA was not ready in ${timeoutMs} ms.`);
```

Comment:
I mean it still makes sense to show lastError.message (if lastError in not null) here as it makes debugging easier

Reply:
other changes are 👍

Reply:
ah, yea it makes sense